samples/generate_html5_console.py

```diff
@@ -69,11 +69,15 @@ def main():
 
     args = get_args()
 
+    context = ssl.SSLContext(ssl.PROTOCOL_TLSv1)
+    context.verify_mode = ssl.CERT_NONE
+
     try:
         si = SmartConnect(host=args.host,
```

Comment:
Thanks! It works.
But I had to login in the web Client using user "administrator@vsphere.local" and password "xxxx..."  to get certificate, at the first time of accessing html5 console in the code using SmartConnectNoSSL on vsphere 6.x and pyvmomi 6.5 release.Is this right?
And, there is bug in line 113 "if vcenter_version <= 5:" as vcenter_version is {str}. It always comes out "false" whenever running code in vsphere 5.x environment.